hw/top_earlgrey/ip_autogen/rv_plic/fpv/tb/waivers.tcl

```diff
@@ -0,0 +1,34 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+# This assertion states that if the input enable (en_i) is not high and onehot input (oh_i) is true
+# then we get an output error (err_o).
+# oh_i is connected to reg_we_check under the instantiation of u_prim_reg_we_check inside u_reg
+# module. Each bit of reg_we_check is assigned through range of different software controlled
+# registers except 6 bits ranging from reg_we_check[186:191] are wired to 0. Those software
+# controlled registers are driven by a addr_hit[x] bit and if that bit is true then reg_addr coming
+# from u_reg_if is the address of that register. Then they also required reg_we=1 and !reg_error.
+# So, if we get a respective addr_hit[x], reg_we and !reg_error then we get reg_we_check as true,
+# which means oh_i is true.
+# en_i is wired to reg_we && !addr_miss expression under the instantiation of u_prim_reg_we_check.
+# If en_i is false, it means that either reg_we is false or addr_miss is true or both. In anycase,
+# we can't get oh_i through the explanation argued above. !addr_miss means that either we don't
+# have a reg_we or the address in reg_addr is not equal to any of the registers (equivalent to
+# !addr_hit).
+cover -disable -regexp ".*\.u_prim_reg_we_check.u_prim_onehot_check\..*\.EnableCheck_A:precondition1"
+
+# This assertion means that if onehot input(oh_i) is not onehot or zero then err_o in
+# u_prim_onehot_check should get asserted. But oh_i can always be a value 1 or 0. The reason is the
```

Comment:
I think that "But oh_i can always be a value 1 or 0" isn't quite the right phrasing. Don't we want to say something like "But oh_i is always one-hot or zero, for the same reason as argued above" ?